src/main/java/net/javadiscord/javabot/systems/commands/ChangeMyMindCommand.java

```diff
@@ -24,14 +24,26 @@
  */
 @Deprecated
 public class ChangeMyMindCommand implements SlashCommand {
+	/**
+	 * The maximum acceptable length for texts to send to the API. Technically,
```

Comment:
What does `f or text` mean?

Reply:
You're reading it wrong. It says "The maximum acceptable length for texts to send to the API"

Reply:
My fault. My device cut that off.